reVX/least_cost_xmission/README.md

```diff
@@ -1,25 +1,145 @@
+* Creating wet and dry (TODO) cost layers
+* Combining wet and dry costs
+* Creating frictions and/or barrier layers
+* Combining friction and barrier layers together
+
+## Config File Keys
+### Required Keys
+The following keys are required to run any type of layer creation:
+
+* `template_raster_fpath` - Template GeoTIFF with the shape, CRS, and transform to use for the project. Data in the raster is ignored.
+* `h5_fpath` - H5 file to store layers in. `h5_fpath` will be created if it does not exist and `existing_h5_fpath` is set.
+
+### Optional Keys
+Most of these keys are optional and affect how the layer creation runs. Some, like `land_mask_vector_fname`, may need to be specified once to generate the mask files, and then can be disabled.
+
+* `existing_h5_fpath` - Existing reV H5 file with the same shape as `template_raster_fpath`. Metadata is copied from `existing_h5_fpath` to `h5_fpath`. This must be defined to create `h5_fpath` if it does not exist.
+* `land_mask_vector_fname` - Polygon GeoPackage or shapefile that defines what areas are considered wet and dry. Features in the file are assumed to represent dry land. The boundaries of features are used to determine the landfall cells.
+* `masks_dir` - Directory to store mask GeoTIFFs in. If not set, masks will be stored in the current working directory.
+* `layer_dir` - By default, all GeoTIFFs listed in `barrier_layers` and `friction_layers` are assumed to be full defined paths or are located in the current working directory. The creator will also search for GeoTIFFs in `layer_dir` if it is set.
+* `save_tiff` - Setting this to `true` will result in the creation of GeoTIFFs for intermediary processing steps. This can be useful for QA/QC.
+
+### Action Keys
+The keys below represent layer creation actions. Mostly analyses will need all layers to be created. Individual creation actions can be rerun as needed, e.g. if it is determined that the `dry_costs` need to be adjusted, but the `wet_costs` are acceptable, the `wet_costs` section can be removed from the config file to prevent it from being recalculated and reduce processing time.
+
+* Costs
+  * `wet_costs` - Costs for wet areas, typically oceans and great lakes.
+  * `dry_costs` - This is not yet implemented. Dry costs must be calculated with the legacy method described below and saved in an H5 file.
+  * `combine_costs` - Combine the wet and dry costs and save to H5. This action must be performed if either wet or dry costs have been recalculated.
+* Friction and barriers
+  * `friction_layers` - Friction areas that are less desirable for transmission routing.
+  * `barrier_layers` - Barrier areas that should not have any transmission in the them. Transmission will route through barriers if there is no other possible route.
+  * `merge_friction_and_barriers` - Combine friction and barriers and save as H5. This action must be performed if the friction or barriers have been modified.
```

Comment:
Do you think this should be `combine...` ? Not sure why I chose merge.

Reply:
Totally up to you, they seem equivalent in my mind

Reply:
Leaving as is